shared/src/packages/users/libs/types/user-sign-up-request-dto.type.ts

```diff
@@ -1,6 +1,9 @@
 import { type UserEntity } from './user-entity.type.js';
 
 type UserSignUpRequestDto = Pick<UserEntity, 'phone'> & {
```

Comment:
This is a bad idea because with UserEntity we should rewrite the user.service and add migrations. Also as I know another student is working now with the user.service

Reply:
got it, thanks!